### PR TITLE
Feature/settings new design

### PR DESCRIPTION
**Pull request by pjosh**
## Overview

settings

### CODE REVIEW

File: app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss [hunk 1]
```diff
@@ -1,54 +1,67 @@
...
   background-color: $white;
   padding: rem(25px) rem(25px) rem(10px);
 
   > div {
     margin-bottom: rem(20px);
-    border-bottom: solid 1px $border;
 
     &:last-child {
-      border: 0;
+      margin-bottom: 0;
```
Comment (edbrett) [inline app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss]: This breaks some cases
screen shot 2018-06-14 at 12 12 42

File: app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss [hunk 2]
```diff
@@ -1,54 +1,67 @@
 @import '~styles/settings.scss';
 
 .c-widget-settings {
-  width: rem(250px);
+  width: rem(310px);
```
Comment (edbrett) [inline app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss]: Can we make this 290px? As for iphone 5 screen width is 320px and we have a standard padding of rem(15px) each side?

File: app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss [hunk 3]
```diff
@@ -1,54 +1,67 @@
...
-        font-size: rem(14px);
-        color: $slate;
-      }
+    .text-date {
+      margin: 0 10px;
     }
   }
 
   .canopy-select {
+    display: flex;
```
Comment (edbrett) [inline app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss]: Something going on here. 
screen shot 2018-06-14 at 12 14 36

**Comment by edbrett** [comment]
Also, I just remembered that the design for the locations selector has changed since this was made. There should be a border between the landCategroy selector and the rest. Like we have here:
screen shot 2018-06-14 at 12 39 03

**Comment by pjosh** [comment]
@edbrett I think all the cases are covered now! Thx for your review 🙇

**Comment by edbrett** [comment]
They love it! Great work duuuuude 🌟

**Pull request merged**